guides/setting-up-mosh-on-koding.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Setting up MOSH on Koding.io
+author: tushar-mathur
```

Comment:
you should put your koding username here so we can link to your koding profile automatically.